resources/public/api/conf/1.0/errors.raml

```diff
@@ -157,4 +157,4 @@ traits:
               ruleDuplicateLossAmount:
                 description: A previous amount was submitted for this tax year
                 value:
-                  code: RULE_DUPLICATE_???
\ No newline at end of file
+                  code: RULE_DUPLICATE_???
```

Comment:
Is it worth removing this error for now? It's not currently being used anywhere.

Reply:
It will be used for the Duplicate "Create" scenario, but we don't know the error yet. I see this as a work in progress so I don't mind leaving in the error - but I also don't mind it being removed. thoughts?

Reply:
Fair enough, I'm happy to leave it in in that case. Will merge.